test/support/api/teams.js

```diff
@@ -50,6 +50,7 @@ export function getTeams({ attributes, relationships, meta } = {}, { depth = 0 }
 // Exporting only teams needed for testing variance
 export const teamCoordinator = getTeamResource({ id: '11111' });
 export const teamNurse = getTeamResource({ id: '22222' });
+export const teamPhysician = getTeamResource({ id: '44444' });
```

Comment:
Can we change the tests to use `teamOther` I don't think should ever need more than 3.

Reply:
Yep, switched this 👍